notebooks/20220512_explore_16s.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>What is the difference between these two plots? One dataset is called phylum and one is called class but the labels are the same, and there's a huge shift from gammaproteobacteria -&gt; alphaproteobacteria. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-cf-sputum/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>one is with taxa aggregated at the class level, and the other is with taxa aggregated at the phylum level. </p><p> </p><p>Taxonomy goes:</p><p>kingdom, phylum, class, order, family, genus, species, strain</p><p> </p><p>16s can only get down to the ~genus level, but sometimes not even that far. Aggregating at different levels gives you different resolution as to what is present. </p><p> </p><p>This is a Very Bad Quality dataset, so everything is mostly labelled as unknown, which isn't really helpful. I left both plots in though because I think they do a nice job of demonstrating some of the issues with these samples (ie, many ASVs didn't end up with good annotations).</p>